source/vibe/web/rest.d

```diff
@@ -762,336 +613,253 @@ private HTTPServerRequestDelegate jsonMethodHandler(alias Func, size_t ridx, T)(
 	return &handler;
 }
 
-/// private
-private string generateRestInterfaceSubInterfaces(I)()
-{
-	if (!__ctfe)
-		assert (false);
 
-	import std.algorithm : canFind;
+string generateRestClientMethods(I)()
```

Comment:
Oh right, I think it ended up as public because I reverted my initial attempt to move the client code to an internal module.